runtime/src/main/java/io/quarkiverse/fury/FuryBuildTimeConfig.java

```diff
@@ -15,4 +20,16 @@ public interface FuryBuildTimeConfig {
     /** Whether to use thread safe fury. The default is true. */
     @WithDefault("true")
     boolean threadSafe();
+
+    /**
+     * Configurations
```

Comment:
Could we add doc for key and value

Reply:
They are in `FuryRegisterClassConfig.java`